pkg/cmd/experimental/syncgroups/groupdetector/groupdetector.go

```diff
@@ -76,13 +80,17 @@ type CompoundDetector struct {
 }
 
 func (l *CompoundDetector) Exists(ldapGrouUID string) (bool, error) {
-	conclusion := false
+	if len(l.locators) == 0 {
+		return false, nil
+	}
+
+	conclusion := true
 	for _, locator := range l.locators {
 		opinion, err := locator.Exists(ldapGrouUID)
 		if err != nil {
 			return false, err
 		}
-		conclusion = conclusion || opinion
+		conclusion = conclusion && opinion
```

Comment:
> can't we return false early if one returns false?

Doc on the type guarantees an error in one errors.  I'm ok with forcing all successful inquiries to be sure about the decision